src/components/Tooltip/index.tsx

```diff
@@ -74,12 +105,13 @@ export const Tooltip = ({
           medium,
           "no-clipboard": noClipboard,
         })}
+        onMouseOver={onMouseOver}
       >
         {children || content}
         {!noClipboard && (
           <span className="clipboard">
             <CopyToClipboard text={content} onCopy={onCopySuccess}>
-              <Icon name="icon icon-clipboard" />
+              <Icon name="icon fa-icon-clipboard" />
```

Comment:
please use icon-clipboard. it won't be rendered if we use fa-icon-clipboard

Reply:
updated, nice catch, thank you